### ISSUE TITLE
Any desire to make this a Meteor-free npm?

### ISSUE THREAD

**Issue by zeroasterisk** [themeteorchef/bert#25]
I love this alert set!  

Do you have any desire to move this away from Meteor package and make it into a npm package?  Perhaps react/redux specific... perhaps not.  

Thanks

**Comment by themeteorchef** [comment]
Thanks :) I do! I'm just a bit gobbed up at the moment. Ideally I'd like to make it dependency free and just based on JS/CSS so anyone can use it (i.e., eliminate usage of Session).

**Comment by zeroasterisk** [comment]
yeah - I could see that happening, since the surface area is so minor... very cool.  carry on with your excellent work.  thanks!

**Comment by NoahVlncrt** [comment]
I've found another little library similar to this. It is also just as elegant looking and it's not dependant on meteor. All you need is some good old fashion npm!
https://github.com/jaredreich/notie
